ui-components/App.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import {
+  DocumentReaderApi,
+  Configuration,
+  Scenario,
+  InlineResponse2001,
+  ProcessRequest,
+} from '@regulaforensics/document-reader-webclient';
+import { ResultTabs } from './components/ResultTabs';
+import { Menu } from './components/Menu';
+import './styles.css';
+
+export const App = () => {
+  const [base64, setBase64] = useState('');
+  const [response, setResponse] = useState<InlineResponse2001>(null);
+  const [request, setRequest] = useState<ProcessRequest>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const showComponent = isLoading || (!isLoading && response);
+
+  useEffect(() => {
+    if (!base64) return;
+
+    const configuration = new Configuration({ basePath: 'https://nightly-api.regulaforensics.com' });
+    const api = new DocumentReaderApi(configuration);
+    const request = {
+      images: [ base64 ],
+      processParam: {
+        scenario: Scenario.FULL_PROCESS,
```

Comment:
```suggestion
        scenario: Scenario.FULL_PROCESS,
        alreadyCropped: true,
```

It needs for the sample images only as they are already cropped by document boundaries